modules/objdetect/src/hog.cpp

```diff
@@ -163,8 +163,10 @@ bool HOGDescriptor::read(FileNode& obj)
     FileNode vecNode = obj["SVMDetector"];
     if( vecNode.isSeq() )
     {
-        vecNode >> svmDetector;
+        std::vector<float> _svmDetector;
+        vecNode >> _svmDetector;
         CV_Assert(checkDetectorSize());
+        setSVMDetector(_svmDetector);
```

Comment:
Looks like, `checkDetectorSize()` check should be after `setSVMDetector()`. Right?

Reply:
@alalek could you remove `CV_Assert(checkDetectorSize());` here because the same line in `setSVMDetector`

Reply:
Yes, we can remove this check.